node/node_cross_shard.go

```diff
@@ -47,7 +47,7 @@ func (node *Node) BroadcastCXReceipts(newBlock *types.Block, lastCommits []byte)
 		if i == int(myShardID) {
 			continue
 		}
-		go node.BroadcastCXReceiptsWithShardID(newBlock, commitSig, commitBitmap, uint32(i))
+		node.BroadcastCXReceiptsWithShardID(newBlock, commitSig, commitBitmap, uint32(i))
```

Comment:
why remove go

Reply:
because there is another go routine inside BroadcastCXReceiptsWithShardID for the p2p sending.